contextualized/regression.py

```diff
@@ -358,10 +358,11 @@ def forward(self, C, T):
 
 
 class ContextualizedRegressionBase(pl.LightningModule):
-    def __init__(self, *args, learning_rate=1e-3, link_fn=linear_fn(), **kwargs):
+    def __init__(self, learning_rate=1e-3, link_fn=linear_fn(), loss_fn=MSE, *args, **kwargs):
```

Comment:
Tests added in #60 and passing (`contextualized/regression/tests.py`). Requires passing dimensions by keyword rather than position, but I think that is okay. What do you think?